src/s2/s2region_term_indexer_test.cc

```diff
@@ -44,10 +45,13 @@ S2_DEFINE_int32(iters, 400, "number of iterations for testing");
 
 namespace {
 
-enum QueryType { POINT, CAP };
+enum DataType {
+  POINT = 0,
```

Comment:
Leave it like before. Explicit = 0 not required.

Reply:
Ok

Reply:
Done